src/AddTodoForm.js

```diff
@@ -1,26 +1,31 @@
 import React, { useState } from "react";
-function AddTodoForm ({onAddTodo}){
-    const [todoTitle, setTodoTitle ] = useState ("") 
-    const handleTitleChange = (event)=> {
-        setTodoTitle(event.target.value) 
-    }
-    const handleAddTodo = (event) => {
+import InputWithLabel from "./InputWithLabel"
 
+function AddTodoForm (props){
+    const [todoTitle , setTodoTitle]= useState("") 
+    const [todos, setTodos] = useState([])
```

Comment:
Hmm not sure you need this anymore since you lifted up the state to the `App.js`